utils.js

```diff
@@ -0,0 +1,29 @@
+const pickEditSchema = (req, res, next) => {
+    const question = require(`./questions/${req.params.id}.js`)
```

Comment:
Could someone break out of the folder by passing an ID that has ../../ in it? Potentially loading a file we don't expect them to load?

Reply:
Yeah, good question

Reply:
Although, the slashes would be hard to pass in considering how urls work

Reply:
oh shoot that got left over and shouldn't be there! thanks for catching!